### PR TITLE
[Flutter] Fix relative path of packages

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR addresses relative path issues on `melos.yaml` & `README.md` of Flutter packages.

CI catches any issues on Dart/Flutter packages properly again.

Made additional changes to prepare packages for publishing.
- Mention package version on `CHANGELOG.md`
- Removed obsolete `LICENSE-APACHE` symlink

Other changelist:
- Make dart output directory if it does not exists
- Replace `continue-on-error`
  - Resume even if previous step fails, this approach displays errors on CI summary.
- Install `Protoc` as part of CI workflow 
  - Added `init-sdk` as part of `just bootstrap` recipe which installs `protobuf`
  - Set version ceiling to `freezed`, `<=2.5.2`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # Breez Liquid SDK
 
-- `lib`: [lib/README.md](lib/core/README.md)
-- `cli`: [cli/README.md](cli/README.md)
-- Bindings
-  - `python`: [lib/bindings/README.md]()
+- `lib`: [lib/README.md](../../lib/core/README.md)
+- `cli`: [cli/README.md](../../cli/README.md)
+- Bindings: [bindings/README.md](../../lib/bindings/README.md)
```
Comment (dangeross) [inline README.md:5]: This README is in the root, so doesn't need the `../../`

Comment (dangeross) [reply README.md:5]: I think you mean't the `packages/dart/README.md` file

Comment (erdemyerebasmaz) [reply README.md:5]: Yes, it was symlinked..

Comment (erdemyerebasmaz) [reply README.md]: @dangeross
Reverted as part of [Update README of Dart/Flutter packages](https://github.com/breez/breez-liquid-sdk/pull/353/commits/97c4b386cb01ee331ae814def3d0e3d6b35bf0b8).

**Review by dangeross — APPROVED** [review]

**Review by dangeross — APPROVED** [review]

**Pull request merged**